### PR TITLE
Add CIAB cert issue info, formatting

### PR DESCRIPTION
**Pull request by rob05c**
#### What does this PR do?

Add CIAB cert issue info, formatting

#### Which TC components are affected by this PR?

- [x] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### Check all that apply

- [ ] This PR includes tests
- [ ] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [ ] This PR includes a database migration (ensure that migration sequence is correct)
- [ ] This PR fixes a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

**Comment by ocket8888** [comment]
Should these maybe also be in the actual docs? I don't remember why I put them in the readme but not in the docs...

File: infrastructure/cdn-in-a-box/README.md [hunk 1]
```diff
@@ -158,8 +158,14 @@ show you the default UI for interacting with the CDN - Traffic Portal.
...
-> If you scroll back through the output ( or use `docker compose logs trafficops-perl | grep "User defined signal 2"` ) and see a line that says something like `/run.sh: line 79: 118 User defined signal 2 $TO_DIR/local/bin/hypnotoad script/cdn` then you've hit a mysterious known error. We don't know what this is or why it happens, but your best bet is to send up a quick prayer and restart the stack.
+### Everything's "waiting for Traffic Ops" forever and nothing seems to be working
 
-> I'm seeing a failure to open a socket and/or set a socket option
-> Try disabling SELinux or setting it to 'permissive'. SELinux hates letting containers bind to certain ports. You can also try re-labeling the `docker` executable if you feel comfortable.
+If you scroll back through the output ( or use `docker compose logs trafficops-perl | grep "User defined signal 2"` ) and see a line that says something like `/run.sh: line 79: 118 User defined signal 2 $TO_DIR/local/bin/hypnotoad script/cdn` then you've hit a mysterious known error. We don't know what this is or why it happens, but your best bet is to send up a quick prayer and restart the stack.
```
[the remark this answers is not in the document]
Comment (rob05c) [reply infrastructure/cdn-in-a-box/README.md]: Fixed

**Comment by rob05c** [comment]
@ocket8888 That would probably be good too, but I'd really like to see it here. I always check Github first, and I'd guess any random person on the internet will as well.

**Comment by ocket8888** [comment]
Oh, of course. I just meant in addition to the README. As a general rule it makes me uncomfortable to have a brief README contain information that isn't available in the official documentation.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3078/
Test PASSed.

**Pull request merged**